docker-compose.test.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
```

Comment:
I think having a proper development Docker setup should make running tests pretty easy, without a separate Docker compose. Running the tests itself should be as easy as `docker-compose exec wca_on_rails bin/rspec`, it's just a matter of putting the few setup instructions somehow. That's just a thought for now ^^

Reply:
Yeah I think you're right here, I'll remove the .test.yml since that does add complexity without much benefit over your solution